### ISSUE TITLE
Earn a loss after the enemy team leaves if they have the lead 

### ISSUE THREAD

**Issue by jannicck** [Bungie-net/api#450]
Hey, 

since the release of the D2 API there is a big bug in ALL game modes! 
If your team behind in a game and the opponend team leave the game you earn a loss on all 3th Party sites! But only on the sites, because  ingame you earn a win on your trials-card,Trials/Competitiv/Quickplay-Win Emblem! 

 Can you please fix that. Its a big issue for teams and players like me, who like to play with friends for stats, because if you play 5 hrs for a win streak and you find a team who do that 1-0 shit your work is broken.... 5hrs for nothing. its a new trend and many players haven´t the motivation to play for a streak and wasting time... 
I have here a few examples: 

Screenshots: 
![image](https://user-images.githubusercontent.com/37842951/37979300-8da2d5b4-31e8-11e8-9169-0b9255d0ee2f.png) , 
![image](https://user-images.githubusercontent.com/37842951/37979343-a789fe6c-31e8-11e8-9509-4849006aef33.png) , 

I hope you respond to me because it makes no more fun to play Trials with the feeling that i can lose every game by cheaters. 

Thank you for your time !

**Comment by bladefist** [comment]
@vthornheart-bng I reported this to you a couple months ago. Do you know if the stats team had a chance to look at it?

thanks!

**Comment by vthornheart-bng** [comment]
Yes, I recall your initial report @bladefist - I have not heard back on the issue yet, and now I can't seem to find the issue in Github, but I remember us talking about this.  I'll hit some folks up and forward them here.  If you all have any additional info, add it to this bug and we'll treat this as the area for continued diagnosis and updates.

**Comment by vthornheart-bng** [comment]
Ah, I see what happened - no Github bug, but I have a log of our prior conversation on it that I just found.  I went back to my initial emails on this subject, and it ended up going to security folks rather than stats (as initially it sounded like it was a DDOS-related in-game bug).  I need to get this information to the Stats team instead.

**Comment by vthornheart-bng** [comment]
TFS 632169, it's on the right people's plate now!

**Comment by vthornheart-bng** [comment]
Is anyone still experiencing this issue recently?

**Comment by vtubelsky** [comment]
Happened to me in Showdown the week it was on.

**Comment by vthornheart-bng** [comment]
@vtubelsky could you send me a PGCR ID of one of the games in question?  It'll help for our investigation!

**Comment by vtubelsky** [comment]
1735662636

**Comment by jannicck** [comment]
Yo, it happens to a friend in trials on saturday so its not fixed and people still do that shit.. if you want i can send you proofs.
